community-membership.md

```diff
@@ -93,6 +93,27 @@ code reviews and work towards becoming an *approver* for the subproject that
 they are active in.  Acceptance of code contributions requires at least one
 approver in addition to the reviews by *members.*
 
+## Triager
```

Comment:
please update the wording in Approvers role:

> Approver status can be scoped to a part of the codebase. For example, critical
core components may have higher bar of becoming an approver. Some approvers may	core components may have higher bar of becoming an approver. Some approvers may
only be doing issues triage and have no approval rights.	only be doing issues triage and have no approval rights.

Reply:
Ok will do.